datalad/tests/utils.py

```diff
@@ -454,7 +454,7 @@ def newfunc(*args, **kwargs):
 
 
 @optional_args
-def with_tempfile(t, content=None, **tkwargs):
+def with_tempfile(t, **tkwargs):
```

Comment:
we should use `@borrow_kwargs` here to place kwargs from `make_tempfile` in this docstring as well... try please